sql/updates/world/master/RoG_2017_06_13_00_world.sql

```diff
@@ -0,0 +1,8 @@
+/* 
+	Spell ID:		81749
+	Spell Name:		Atonement
+	Spell Family: 	SPELLFAMILY_PRIEST
+*/
```

Comment:
3 changes needed in this file:
1. add NewLine at End of File (currently missing)
2. never use apostrophes around integer number values (spell_id)
3. TC codestyle: use single line `--` instead of `/* */` for short comments. ( `/* */` is for GPL headers and long explanations in big files.)